sos/plugins/openstack_neutron.py

```diff
@@ -61,10 +60,10 @@ def postproc(self):
             "ssl_key_password", "nsx_password", "vcenter_password",
             "edge_appliance_password", "tenant_admin_password", "apic_password"
         ]
-        regexp = r"((?m)^\s*#*(%s)\s*=\s*)(.*)" % "|".join(protect_keys)
+        regexp = r"((?m)^\s*(%s)\s*=\s*)(.*)" % "|".join(protect_keys)
```

Comment:
When doing a change here, could you also please remove duplicate `qpid_password` in `protected_key` (1st and 3rd line of the list)?